lib/registries/addon/components/review-actions-list/review-action/component.ts

```diff
@@ -0,0 +1,47 @@
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import Intl from 'ember-intl/services/intl';
+
+import { RegistrationReviewStates } from 'ember-osf-web/models/registration';
+import ReviewActionModel, { ReviewActionTrigger } from 'ember-osf-web/models/review-action';
+import formattedTimeSince from 'ember-osf-web/utils/formatted-time-since';
+
+interface Args {
+    reviewAction: ReviewActionModel;
+    embargoEndDate: Date;
+}
+
+export default class ReviewAction extends Component<Args> {
+    @service intl!: Intl;
+
+    get hasComment() {
+        return Boolean(this.args.reviewAction);
+    }
+
+    get translationString() {
+        const { reviewAction } = this.args;
+        if (reviewAction.actionTrigger === ReviewActionTrigger.AcceptSubmission) {
+            if (reviewAction.toState === RegistrationReviewStates.Embargo) {
+                return this.intl.t('registries.reviewAction.acceptEmbargoSubmission',
+                    {
+                        action: reviewAction.triggerPastTense,
+                        moderator: reviewAction.creator.get('fullName'),
```

Comment:
The creator of a `review-action` is not always a moderator. For example, a `submit` review action will have the submitter of the registration as the creator.
This doesn't break functionality though, because the translation strings doesn't care about whether the creator is a moderator or not. But might make sense to rename this context var to something else.